services/movie.py

```diff
@@ -0,0 +1,36 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None) -> QuerySet:
+    if not genres_ids and not actors_ids:
+        return Movie.objects.all()
+    if genres_ids and actors_ids:
+        return Movie.objects.filter(
+            genres__in=genres_ids).filter(actors__in=actors_ids)
+    if actors_ids and not genres_ids:
+        return Movie.objects.filter(actors__in=actors_ids)
+    if genres_ids and not actors_ids:
+        return Movie.objects.filter(genres__in=genres_ids)

```

Comment:
You should simplify your code. Now, you're using 4 `if` statements, but you can realize it with using 2 `if` statements
```suggestion
    queryset = Movie.objects.all()
    if actors_ids:
        queryset = queryset.objects.filter(actors__in=actors_ids)
    if genres_ids:
         queryset = queryset.objects.filter(genres__in=genres_ids)
    return queryset
```